rollup.config.mjs

```diff
@@ -0,0 +1,51 @@
+import {nodeResolve} from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import json from "@rollup/plugin-json";
+import alias from "@rollup/plugin-alias";
+import terser from "@rollup/plugin-terser";
+import typescript from "@rollup/plugin-typescript";
+import nodePolyfills from "rollup-plugin-polyfill-node";
+import pkg from "./package.json" assert {type: 'json'};
+
+export default [
+  // Browser
+  {
+    input: "yajsapi/index.ts",
+    output: {
+      inlineDynamicImports: true,
+      file: pkg.browser,
+      name: "yajsapi",
+      sourcemap: true,
+      format: "es"
+    },
+    plugins: [
+      alias({
+        entries: [
+          {find: "stream", replacement: "stream-browserify"},
+          {find: /\.\/pinoLogger.js/, replacement: "tests/mock/utils/empty.js"},
+          {find: /RedisDatastore/, replacement: "tests/mock/utils/empty_default.js"},
+          {find: /IORedisConnection/, replacement: "tests/mock/utils/empty_default.js"},
+          {find: /RedisConnection/, replacement: "tests/mock/utils/empty_default.js"},
+          {find: /eventsource\/lib\/eventsource.js/, replacement: "tests/mock/utils/empty_default.js"},
+          {find: /src\/api\/provider-api$/, replacement: "."},
+          {find: /\.\/gftp.js/, replacement: "tests/mock/utils/empty.js"},
+        ],
+      }),
+      nodeResolve({browser: true, preferBuiltins: true}),
+      commonjs(),
+      nodePolyfills(), // ToDo Needed?
```

Comment:
Deal with todos, @mgordel , can you provide me with more context for some of those? Why we had to introduce them? Can some of them be dropped?

Reply:
long term I think we should drop stream support and move to native browser stuff with polyfills.

if some native node functionality is needed, we make a separate lib for it